backstop.json

```diff
@@ -0,0 +1,82 @@
+{
+  "viewports": [
+    {
+      "label": "desktop",
+      "width": 1280,
+      "height": 1080
+    }
+  ],
+  "onBeforeScript": "puppet/onBefore.js",
+  "onReadyScript": "puppet/onReady.js",
```

Comment:
These 2 lines were copied in from an example config file, I think we can remove?  

```
CREATING NEW REFERENCE FILE
WARNING: script not found: /Users/rachelwhitton/documentation/backstop_data/engine_scripts/puppet/onBefore.js
Browser Console Log 0: JSHandle:BackstopTools have been installed.
Browser Console Log 0: JSHandle:BackstopTools have been installed.
Browser Console Log 0: JSHandle:BackstopTools have been installed.
Browser Console Log 0: JSHandle:BackstopTools have been installed.
Browser Console Log 0: JSHandle:BackstopTools have been installed.
Browser Console Log 0: JSHandle:BackstopTools have been installed.
Browser Console Log 0: JSHandle:BackstopTools have been installed.
Browser Console Log 0: JSHandle:BackstopTools have been installed.
WARNING: script not found: /Users/rachelwhitton/documentation/backstop_data/engine_scripts/puppet/onReady.js
```